Makefile.core.mk

```diff
@@ -347,6 +347,11 @@ gen-check: check-no-modify gen check-clean-repo
 # Copy the injection template file and configmap from istiod chart to istiod-remote chart
 sync-configs-from-istiod:
 	cp manifests/charts/istio-control/istio-discovery/files/injection-template.yaml manifests/charts/istiod-remote/files/
+
+	# Copy over values, but apply some local customizations
+	cp manifests/charts/istio-control/istio-discovery/values.yaml manifests/charts/istiod-remote/
+	yq w manifests/charts/istiod-remote/values.yaml telemetry.enabled false -i
```

Comment:
This doesn’t seem maintainable. Is there a better way to provide/merge overrides.

Reply:
There is `yq merge` but I could n't get it working

Reply:
I just meant how will anyone know which values to merge/override?

Reply:
I don't know, any suggestions? This is no different than current code btw, just more explicit. we probably need to clean up the charts given the new multicluster structure

Reply:
Not really. It's better than having two full copies I guess.